Bottom: 8d01eb93e5776cbe5a903f0de135da0d92a3dca7
Top:    331ad50fdba30394cace3dfdd9768c96481d9434
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 14:54:36 +0000

Refresh of more-play-with-quad-edge-data

---

diff --git a/src/hid/gtk/contour3d.c b/src/hid/common/contour3d.c
similarity index 100%
rename from src/hid/gtk/contour3d.c
rename to src/hid/common/contour3d.c
diff --git a/src/hid/gtk/contour3d.h b/src/hid/common/contour3d.h
similarity index 100%
rename from src/hid/gtk/contour3d.h
rename to src/hid/common/contour3d.h
